examples/cpu_temperature.py

```diff
@@ -0,0 +1,46 @@
+import colorsys
+import time
+
+import openrazer.client
+
+devman = openrazer.client.DeviceManager()
+
+# Show temperature colour on this device
+device = None
+for d in devman.devices:
+    if d.name == "Razer BlackWidow Chroma":
```

Comment:
```suggestion
    if "Razer BlackWidow Chroma" in d.name:
```

Reply:
I was on the fence with this change from `==` (exactly equal) to `in` (substring match) but I suppose it might be useful.